src/fragment-components/accordion/a-accordion-item.tsx

```diff
@@ -136,13 +136,13 @@ export const componentInfo: ComponentInfo = {
 	image,
 	codeExport: {
 		angular: {
-			inputs: ({ json }) => `@Input() ${nameStringToVariableString(json.codeContext?.name)}Title = "${json.title}";`,
+			inputs: ({ json }) => `@Input() ${nameStringToVariableString(json.codeContext?.name)}Title = "${json.title}";
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}Disabled = ${json.disabled};`,
```

Comment:
Value for disabled is undefined if user does not interact with the disabled checkbox. Add a condition here to check if  `json.disabled` value is undefined: if so, false, else true.

You can also set initial value for `disabled` in `componentInfo`, see other components for example.